Compiler/FrontEnd/ClassLoader.mo

```diff
@@ -525,18 +530,32 @@ algorithm
 end makeClassLoad;
 
 protected function checkPackageOrderFilesExist
-  input PackageOrder po;
+  input output PackageOrder po;
   input String mp;
   input SourceInfo info;
   input Boolean encrypted = false;
+  input output list<String> differences;
 algorithm
   _ := match (po,mp,info)
     local
-      String pd,str;
+      String pd,str,str2,str3,str4;
+      list<String> strs;
     case (CLASSLOAD(str),_,_)
-      equation
-        pd = Autoconf.pathDelimiter;
-        Error.assertionOrAddSourceMessage(System.directoryExists(mp + pd + str) or System.regularFileExists(mp + pd + str + (if encrypted then ".moc" else ".mo")),Error.PACKAGE_ORDER_FILE_NOT_FOUND,{str},info);
+      algorithm
+        pd := Autoconf.pathDelimiter;
+        str2 := str + (if encrypted then ".moc" else ".mo");
+        if not (System.directoryExists(mp + pd + str) or System.regularFileExists(mp + pd + str2)) then
+          try
+            str3 := List.find(System.moFiles(mp), function Util.stringEqCaseInsensitive(str2=System.tolower(str2)));
```

Comment:
The `tolower()` call is not needed for `stringEqCaseInsensitive()` as it does the `tolower()` itself.

Reply:
True...